src/v/cluster/rm_stm.cc

```diff
@@ -2529,8 +2529,8 @@ rm_stm::apply_snapshot(stm_snapshot_header hdr, iobuf&& tx_ss_buf) {
         const auto pid = entry.pid;
         auto it = _log_state.seq_table.find(pid);
         if (it == _log_state.seq_table.end()) {
-            _log_state.seq_table.try_emplace(
-              it, pid, seq_entry_wrapper{.entry = std::move(entry)});
+            seq_entry_wrapper seq{std::move(entry)};
+            _log_state.seq_table.try_emplace(it, pid, std::move(seq));
```

Comment:
just to confirm, these two lines of change aren't involved in the fix, right? they appear to have the same semantics before/after.

Reply:
ya correct, I made the constructor explicit, so had to make this change.